capstone/src/main/webapp/index.html

```diff
@@ -15,6 +15,11 @@ <h1>Enforcer</h1>
     </div>
     <div class="clear"></div>
     <div id="content" class="content">
+    <form method="POST" action="/sentiment">
+      <textarea name="message"></textarea>
+      <br/>
+      <button>Submit</button>
```

Comment:
Quick question, what is this submit button for?

Reply:
for a form I was using to test sentiment, I'm deleting it now